ui/app/AppMain.qml

```diff
@@ -57,12 +57,14 @@ Item {
 
         if (localAccountSensitiveSettings.onlyShowContactsProfilePics) {
             const isContact = appMain.rootStore.contactsModuleInst.model.list.rowData(index, "isContact")
-            if (isContact === "false") {
-                return
+            if (isContact === "true") {
+                return appMain.rootStore.contactsModuleInst.model.list.rowData(index, useLargeImage ? "largeImage" : "thumbnailImage")
```

Comment:
I don't think you needed to change this. The line below (65) should work to grab the image in the case of a contact as well.

Reply:
This comment still stands, you can remove that if and that line since line 67 does the job